compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmPackageGen.java

```diff
@@ -452,7 +477,7 @@ CompiledJarFile generate(BIRNode.BIRPackage module, InteropValidator interopVali
 
         // generate object/record value classes
         JvmValueGen valueGen = new JvmValueGen(module, this, methodGen, lambdaGen);
-        valueGen.generateValueClasses(jarEntries);
+        valueGen.generateValueClasses(jarEntries, moduleInitClass);
```

Comment:
I have been wondering if there could be a better way rather than having to pass this parameter to many methods